app/views/posts/index.html.erb

```diff
@@ -21,7 +21,7 @@
       </div>
   </div>
 
-  <% @posts.each do |n| %>
+  <% @posts.includes(:comments).each do |n| %>
```

Comment:
Thanks for the review. I used the 'include' for eager loading to also load the comments when I rendered the posts in views/posts/index.html.erb line 24